src/Microsoft.Sbom.Adapters/Adapters/ComponentDetection/SpdxComponentExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.ComponentDetection.Contracts.TypedComponent;
+using Microsoft.Sbom.Contracts;
+using Microsoft.Sbom.Contracts.Enums;
+
+namespace Microsoft.Sbom.Adapters.ComponentDetection
+{
+    /// <summary>
+    /// Extensions methods for <see cref="SpdxComponent"/>.
+    /// </summary>
+    internal static class SpdxComponentExtensions
```

Comment:
I thought we already have a mapping for SPDXComponent, isnt't that the case?

Reply:
I can't find any implementation of that, though there is functionality to convert and SpdxComponent to an external reference.  https://github.com/microsoft/sbom-tool/blob/40367fc11a6778d5cbb7da90466a87086ad4be9c/src/Microsoft.Sbom.Api/Converters/ComponentToExternalReferenceInfoConverter.cs#L62